### ISSUE TITLE
Filter for URLs that are duplicate DOIs

### ISSUE THREAD

**Issue by pmbittner** [tthuem/MibTeX#51]
Sometimes, the URL of an entry is just the doi lookup. It would be useful to add an option to filter such URLs in the cleaner (similar to how we can currently filter entire fields currently). Such a filter would be useful for submissions to ACM or DFG which need DOIs but we could save space when omitting duplicate URLs.

**Comment by tthuem** [comment]
Why do we want to add URLs that are just DOIs in the first place? Typically, I remove those when adding new entries.

Still, there are cases (e.g., theses at OPARU) for which we explicitly link to the PDF. So even it we would be so restrictive when to add DOIs there are still cases when we probably do not want two links for a single publication in references.

**Comment by tthuem** [comment]
@pmbittner Would that be a useful check for BibTags' CI?

**Comment by pmbittner** [comment]
Yes, it would be best to eliminate such pseudo URLs in BibTags in the first place! I would propose to close this issue here and reopen it in BibTags, potentially with a pointer to this issue.

**Comment by tthuem** [comment]
Sounds good.
